src/KIM/kim_interactions.cpp

```diff
@@ -129,6 +130,7 @@ void KimInteractions::do_setup(int narg, char **arg)
                     "=======\n");
 
   if (simulatorModel) {
+    auto first_visit = input->variable->find("kim_update");
```

Comment:
@ProfessorMiller this is correct. It would be better to move it after line 202. Other parts of the code do not use this variable, so you can keep it local within the scope.